jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/ConcurrentPool.java

```diff
@@ -597,4 +667,100 @@ public String toString()
                 getPooled());
         }
     }
+
+    public interface Holder<P>
```

Comment:
Should be `private`.

Reply:
I've made protected for now, as it is exposed in my proposed `leaded(Holder<P>)` method for debugging.